src/packages/cli/src/args.ts

```diff
@@ -281,9 +281,17 @@ export default function (
             }
           )
           .version(false);
+      },
+      function () {
+        // this handler executes when `ganache instances` is called without a subcommand
+        const command = chalk`{hex("${TruffleColors.porsche}") ganache instances}`;
+        console.log(`${command} requires a subcommand:`);
```

Comment:
I don't think the colon doesn't makes sense here, mostly because the list of subcommands doesn't come until after some other usage information.

```suggestion
        console.log(`${command} requires a subcommand.`);
```